v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBigQueryFlexOptions.java

```diff
@@ -157,7 +168,7 @@ public interface KafkaToBigQueryFlexOptions
       parentName = "avroFormat",
       parentTriggerValues = {"CONFLUENT_WIRE_FORMAT"},
       optional = true,
-      description = "Naming prefix to be used while creating BigQuery output tables.",
+      description = "Naming prefix to be used while creating BigQuery output tables",
```

Comment:
```suggestion
      description = "Naming prefix to be used while creating BigQuery output tables",
```

Can we make it a little clear saying we are going to prefix it to the BQ table name?

Reply:
Cleared the naming a bit